app/assets/javascripts/discourse/app/lib/sidebar/messages-section/group-message-section-link.js

```diff
@@ -0,0 +1,80 @@
+import I18n from "I18n";
+
+import { tracked } from "@glimmer/tracking";
+import { capitalize } from "@ember/string";
+
+import { INBOX } from "discourse/components/sidebar/messages-section";
+
+export default class GroupMessageSectionLink {
+  @tracked shouldDisplay = this._isInbox ? true : false;
+
+  routeNames = new Set([
```

Comment:
Should this also have `const` or is that not necessary within a class like it's used here?

Reply:
Since this is a [public field declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#field_declarations) we do not require the `const`, `let` or `var` keywords.

Reply:
Cool, thought it might be something like that, thanks.